_layouts/base.html

```diff
@@ -12,13 +12,22 @@
       name="viewport"
       content="width=device-width, initial-scale=1, user-scalable=no"
     />
-    <meta property=”og:title” content="{{ page.title | default: site.title }}" />
-    <meta property=”og:url” content="https://codevid19.com" />
-    <meta property=”og:image” content="https://codevid19.com/images/codevid-19_insta.png" />
-    <meta property="og:description" content="{{ page.description | default: site.description }}" />
-    <link rel="stylesheet" href="assets/css/main.css" />
+    <meta
+      property="”og:title”"
+      content="{{ page.title | default: site.title }}"
+    />
+    <meta property="”og:url”" content="https://codevid19.com" />
+    <meta
+      property="”og:image”"
+      content="https://codevid19.com/images/codevid-19_insta.png"
+    />
+    <meta
+      property="og:description"
+      content="{{ page.description | default: site.description }}"
+    />
+    <link rel="stylesheet" href="/assets/css/main.css" />
     <noscript>
-      <link rel="stylesheet" href="assets/css/noscript.css"
+      <link rel="stylesheet" href="/assets/css/noscript.css"
```

Comment:
These urls shouldn't be pathed to root as that might not always be true depending on how they're served.

Instead jekyll offers a liquid templates filter that can make url's relative and works in collection pages. It's called `relative_url`. Here's an example of a time I used it on the the DES RFC repo:

https://github.com/devedmonton/rfcs/blob/master/_layouts/default.html#L7

Reply:
The example also has a root pathstring, so are you saying for these stylesheet URLs (and the rest of the URLs I changed in this PR), the work I need to put in is wrapping these paths with a `relative_url` filter?